src/main/plugin/iso19139.ca.HNAP/process/fix-date-format.xsl

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0"
+                  xmlns:gco="http://www.isotc211.org/2005/gco"
+                  xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xmlns:geonet="http://www.fao.org/geonetwork"
+                  exclude-result-prefixes="#all">
+
+  <xsl:template match="gmd:MD_DataIdentification/gmd:citation/gmd:CI_Citation/gmd:date/gmd:CI_Date/gmd:date[gco:Date and string-length(gco:Date) != 10]/gco:Date">
```

Comment:
Question?

Is this required for the following as well?
gmd:identificationInfo/*/gmd:extent/gmd:EX_Extent/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent/gml:TimePeriod/gml:beginPosition

We currently have issues when publishing this date to Open Data.  It does not support single date so we have to add the 01-01 as well to the year.  It is possible that this is an Open Data issues and not necessarily an HNAP issue so I'm just asking the question.